src/flutter/shell/platform/linux_embedded/window/linuxes_window_drm.cc

```diff
@@ -97,7 +95,12 @@ bool LinuxesWindowDrm::DispatchEvent() {
 }
 
 bool LinuxesWindowDrm::CreateRenderSurface(int32_t width, int32_t height) {
-  native_window_ = std::make_unique<NativeWindowDrm>(kDrmDeviceFilename);
+  auto device_filename = getenv("DRM_DEVICE_FILENAME");
```

Comment:
Could you change from `DRM_DEVICE_FILENAME` to `FLUTTER_DRM_DEVICE`?

Reply:
Please use `static constexpr char` ...

Reply:
I fixed it.